README.md

```diff
@@ -693,6 +690,49 @@ Updating virtual Go repository:
 err = servicesManager.UpdateVirtualRepository().Go(params)
 ```
 
+#### Creating and Updating Federated Repository
+You can create and update a federated repository for the following package types:
+
+Alpine, Bower, Cran, Cargo, Chef, Cocoapods, Composer, Conan, Conda, Debian, Docker, Gems, Generic, Gitlfs, Go, Gradle, Helm, Ivy, Maven, Npm, Nuget, Opkg, Puppet, Pypi, Rpm, Sbt, Vagrant and Yum
+
+Each package type has it's own parameters struct, can be created using the method
```

Comment:
it's --> its
(looks like this typo was copied from somewhere elase in the README, if you're able to fix all, that would be great :))

Reply:
Done ✔️ (including the other 3 locations)